src/styleguide/mock-data/api/browser.js

```diff
@@ -0,0 +1,8 @@
+import { setupWorker } from 'msw';
```

Comment:
suggestion: I would recommend to lift the `api` folder one level up to `styleguide`.